functions.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	util "github.com/aokoli/goutils"
+	"github.com/huandu/xstrings"
```

Comment:
Since this is imported directly, can you also add it to the `glide.yaml` file? Then I'll get this merged.